model/host/hostutil.go

```diff
@@ -156,7 +156,38 @@ func initSystemCommand() string {
 	`
 }
 
-// FetchJasperCommands builds the command to download and extract the Jasper
+// FetchAndReinstallJasperCommand returns the command to fetch Jasper and
+// restart the service with the latest version.
+func (h *Host) FetchAndReinstallJasperCommand(config evergreen.JasperConfig) string {
+	return strings.Join([]string{
+		h.FetchJasperCommand(config),
+		h.ForceReinstallJasperCommand(config),
+	}, " && ")
```

Comment:
shouldn't this all be able to be jasper.Command? rather than strings?

Reply:
but we could change it to be a command, and have RunSSHCommand do a remote jasper.Command invocation